themes/admin/Sea_Green/css/sea_green.css

```
@@ -2892,22 +2861,22 @@ textarea.form-control-lg {
-  border-color: rgb(43.5, 182.1, 108.3);
+  background-color: #5bc68d;
+  border-color: #2cb66c;
 }
 .btn-check:focus + .btn-primary, .btn-primary:focus {
   color: #000;
-  background-color: rgb(90.5, 198.3, 140.9);
-  border-color: rgb(43.5, 182.1, 108.3);
-  box-shadow: 0 0 0 0.25rem rgba(17, 147.9, 78.2, 0.5);
+  background-color: #5bc68d;
+  border-color: #2cb66c;
+  box-shadow: 0 0 0 0.25rem rgba(17, 148, 78, 0.5);
```

Stylelint (17.2.0)

[error] Expected "0.5" to be "50%" (alpha-value-notation)

(alpha-value-notation)